.clang-format

```diff
@@ -38,56 +60,83 @@ BraceWrapping:
   AfterExternBlock: false
   BeforeCatch:     true
   BeforeElse:      true
+  BeforeLambdaBody: false
+  BeforeWhile:     false
   IndentBraces:    false
   SplitEmptyFunction: false
   SplitEmptyRecord: false
   SplitEmptyNamespace: false
 BreakBeforeBinaryOperators: None
+BreakBeforeConceptDeclarations: Allowed
 BreakBeforeBraces: Custom
 BreakInheritanceList: BeforeColon
 BreakBeforeTernaryOperators: true
 BreakConstructorInitializers: BeforeColon
 BreakStringLiterals: false
 ColumnLimit:     105
 CommentPragmas:  '^ IWYU pragma:'
+QualifierAlignment: Left
 CompactNamespaces: false
-ConstructorInitializerAllOnOneLineOrOnePerLine: false
 ConstructorInitializerIndentWidth: 4
 ContinuationIndentWidth: 4
 Cpp11BracedListStyle: true
 DeriveLineEnding: false
 DerivePointerAlignment: false
 DisableFormat:   false
+EmptyLineAfterAccessModifier: Never
+EmptyLineBeforeAccessModifier: Always
+PackConstructorInitializers: BinPack
 FixNamespaceComments: false
 ForEachMacros:
   - foreach
   - Q_FOREACH
   - BOOST_FOREACH
-IncludeBlocks:   Preserve
+IfMacros:
+  - KJ_IF_MAYBE
+IncludeBlocks:   Regroup
 IncludeCategories:
-  - Regex:           'dlaf.*/'
+  - Regex:           '^<dlaf_test'
+    Priority:        7
+    SortPriority:    0
```

Comment:
Do I understand correctly that `SortPriority` is for sorting of _groups_ of includes, rather than individual includes? And setting them all to `0` means that all includes are going to be regrouped into a single group?

Reply:
Different priority means different group, while `SortPriority` is used to specify the order in the given group.
`Priority: 0` is the special group clang-format uses for the corresponding header

e.g.
* header C: P 1 SP 0
* header B: P 0 SP 0
* header A: P 1 SP 2
will be ordered as
```
#include <B>

#include <C>
#include <A>
```